suites/upgrade/firefly-hammer-x/parallel/7-final-workload/rados_mon_thrash.yaml

```diff
@@ -0,0 +1,10 @@
+tasks:
+   - sequential:
+      - mon_thrash:
+          revive_delay: 20
+          thrash_delay: 1
+      - workunit:
+          clients:
+            client.1:
+            - rados/test-upgrade-v9.0.1.sh
```

Comment:
Sage, I rearranged loads : 
in step 5 running "rados/test"
in step 7 rados/test-upgrade-v9.0.1.sh
This is a bit different what you said first time, so pls confirm.